README.md

```diff
@@ -1,2 +1,100 @@
 # Awesome_Books
 Creation Book List Library for a user to store the list of books 
+
+> A demo website for Book Library.
+
+## Built With
+
+HTML and CSS
+Javascript : Arrays and Objects , DOM , Local Storage etc 
+
+## Video
+
+In the following [link]()
```

Comment:
- [ ] **Optional** There is no link, use _coming soon_ if link isn't available at the moment. I hope it makes sense 😄